core/src/test/java/org/testcontainers/junit/DockerComposeContainerWithBuildTest.java

```diff
@@ -0,0 +1,20 @@
+package org.testcontainers.junit;
+
+import java.io.File;
+import org.junit.Rule;
+import org.testcontainers.containers.DockerComposeContainer;
+
+public class DockerComposeContainerWithBuildTest extends BaseDockerComposeTest {
+
+    @Rule
+    public DockerComposeContainer environment = new DockerComposeContainer(new File("src/test/resources/compose-test.yml"))
+            .withExposedService("redis_1", REDIS_PORT)
+            .withExposedService("db_1", 3306)
+            .withBuild(true)
```

Comment:
Is there anything we can do to actually verify the build and remove options work?

As it is, it seems that we’ll get a failure if the build feature throws an exception, but otherwise there’s no verification.

Reply:
I am not so into the internals of Testcontainers nur one thing which came into my mind was to verify if the image was build freshly by checking the modifcation date of the file? Would this be an option?

Reply:
We could check for the creation date of the image.